scrapy/extensions/feedexport.py

```diff
@@ -160,9 +160,10 @@ def _store_in_thread(self, file):
 
 class GCSFeedStorage(BlockingFeedStorage):
 
-    def __init__(self, uri, project_id, acl):
+    def __init__(self, uri, project_id, acl, timeout):
```

Comment:
I think we could make this an option that you can pass to `FEEDS`, and hence read it from a `feed_options` argument here (see how `FTPFeedStorage` reads the `overwrite` option below).